cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/CloudControllerClient.java

```diff
@@ -1331,4 +1332,12 @@ public void createApplication(String applicationName, Staging staging, Integer d
 
     List<CloudBuild> getBuildsForPackage(UUID packageGuid);
 
+    List<CloudApplication> getApplicationsByMetadata(String labelSelector);
```

Comment:
Provide Metadata as argument or if not applicable, rename the method to be
getApplicationsByMetadaLabelSelector